src/containers/Tenant/QueryEditor/QueriesHistory/QueriesHistory.tsx

```diff
@@ -17,8 +18,7 @@ interface QueriesHistoryProps {
 
 function QueriesHistory(props: QueriesHistoryProps) {
     const [isHistoryVisible, setIsHistoryVisible] = useState(false);
-    const history: string[] =
-        useSelector((state: any) => state.executeQuery.history?.queries) ?? [];
+    const history = useTypedSelector((state) => state.executeQuery.history.queries) ?? [];
```

Comment:
we don't need to check if `history` exists in state?

Reply:
We update history in actions, that are all typed now. And `executeQuery` reducer state is typed too. I hope there is no need in the additional check and typecheck will tell us if anything will be broken